docs/_static/custom.css

```diff
@@ -8,14 +8,34 @@
   -webkit-touch-callout: none;
 }
 
-.admonition-example-code > .admonition-title:after {
+.admonition-example-code>.admonition-title:after {
   font-family: "FontAwesome";
   content: "";
   float: right;
 }
 
-.admonition-example-code > .admonition-title.open:after {
+.admonition-example-code>.admonition-title.open:after {
   font-family: "FontAwesome";
   content: "";
   float: right;
 }
+
+.genindextable {
+  display: block;
+  width: 100%;
+}
+
+table.genindextable tbody {
+  display: block;
+  width: 100%;
+}
+
+table.genindextable tbody tr {
+  display: block;
+  width: 100%;
+}
+
+table.genindextable tbody tr td {
+  display: block;
+  width: 100% !important;
+}
```

Comment:
nit: this was to fix a long-standing bug in the documentation where generated index had two columns, and the 2nd one was rendered outside of the theme. With this, everything is in a single column now.